### ISSUE TITLE
Drupal: color contrast

### ISSUE THREAD

**Issue by rschwab** [UCSCLibrary/SiteImprove#100]
Found insufficient color contrast on a few pages:

- [x] button at https://library.ucsc.edu/news/are-you-supported-by-eop-drc-or-are-you-military-connected
- [x] button at https://library.ucsc.edu/news/are-you-supported-by-eop-drc-or-are-you-military-connected
- [x] The news page where the buttons above appear
- [ ] The giving page "Campaign for UCSC" https://library.ucsc.edu/giving-to-the-library

Screenshot of buttons:
<img width="1071" height="561" alt="Image" src="https://github.com/user-attachments/assets/3a8f4d90-6e1d-438a-8d6e-20ed0e11b070" />

**Comment by rschwab** [comment]
I'll email Linda about this text. Can be fixed by changing to hex color #967103

<img width="928" height="288" alt="Image" src="https://github.com/user-attachments/assets/e3e339fa-3ebb-4574-b7cb-b6dde1ee0b0a" />

**Comment by rschwab** [comment]
@JessWaggs Can I assign the buttons to you? Here's a screenshot with #20809D, a blue with the 4.5:1 ratio to white text.

<img width="746" height="297" alt="Image" src="https://github.com/user-attachments/assets/b339d0d2-cd96-417b-b597-061b171e0bff" />

**Comment by JessWaggs** [comment]
@rschwab I just unpublished that news item, so it's fixed! :) 

But that button just used the standard button class class="btn btn-info btn-lg button" – our new design system uses a darker button, so I think this will be part of the upcoming sitewide styling revision to update the color associated with this class?

**Comment by JessWaggs** [comment]
There was a button with the same styling on [this news page](https://library.ucsc.edu/news/library-local-subscription-impact-survey), so I added some inline styling to make it the dark blue in our new design system.

**Comment by rschwab** [comment]
Thanks Jess, Linda updated the giving page so this one is resolved. Siteimprove hasn't caught up yet, but we're back at 100% A/AA
